man/timesyncd.conf.xml

```diff
@@ -101,6 +101,36 @@
         is used instead.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>MinPollIntervalSec=</varname></term>
+        <term><varname>MaxPollIntervalSec=</varname></term>
+        <listitem><para>The minimum and maximum poll intervals for NTP messages.
+        Take time (in seconds). <varname>MinPollIntervalSec=</varname> must be larger than 16 sec.
+        <varname>MaxPollIntervalSec=</varname> must be larger than <varname>MinPollIntervalSec=</varname>.
+        <varname>MinPollIntervalSec=</varname> defaults to 32 seconds, and
+        <varname>MaxPollIntervalSec=</varname> defaults to 2048 seconds.</para></listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><varname>Accuracy=</varname></term>
+        <listitem><para>Expected Accuracy of time synchronization in seconds, which is used to adjust the
+        poll interval. Takes a positive floating point value. Defaults to 0.2.</para></listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><varname>MaxAdjust=</varname></term>
+        <listitem><para>Maximum delta in seconds which the system clock is gradually adjusted (slew) to approach
+        the network time. Deltas larger than this are set by letting the system time jump. The kernel's limit
+        for adjtime is 0.5 seconds. Takes a positive floating point value less than 0.5.
+        Defaults to 0.4 seconds.</para></listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><varname>MaxRootDistance=</varname></term>
+        <listitem><para>Maximum acceptable root distance in seconds.
```

Comment:
if this is a value in the unit of seconds, then this should be called RootDistanceMaxSec= i figure, and not take a floating point value, but the usual time value, at least facing towards user configuration